test/integration.test.js

```diff
@@ -74,6 +74,27 @@ describe('Page', () => {
       }
     });
 
+    describe('Feeds', () => {
+      it(`fetch an XML feed`, (done) => {
```

Comment:
it fetch? :)

The "it" blocks are for expectations ("it does xyz"). "describe" is for descriptions.